descriptors/ModuleDescriptor-template.json

```diff
@@ -410,6 +421,11 @@
       "displayName": "Data Export - get mapping transformation fields collection",
       "description": "Entry point to get mapping transformation fields for the fields"
     },
+    {
+      "permissionName": "ddata-export.error-logs.collection.get",
```

Comment:
Please correct the typo in permission name

Reply:
Also it must be *.item.get. We are getting the logs per ID, not the entire set of logs present